packages/solid-start-client/src/StartClient.tsx

```
@@ -1,37 +1,15 @@
-import { Await, HeadContent, RouterProvider } from '@tanstack/solid-router'
-import { hydrateStart } from '@tanstack/start-client-core/client'
+import { createResource, Suspense } from 'solid-js';
```

ESLint

[error] Member 'Suspense' of the import declaration should be sorted alphabetically.

(sort-imports)